core/state/node/migrations/0000000005-web-driver.json

```diff
@@ -0,0 +1,26 @@
+{
+    "old_version": "v0.0.4",
+    "new_version": "v0.0.5",
+    "up": [
+        {
+            "op": "add",
+            "path": "/properties/app_installations/additionalProperties/properties/driver/enum/1",
```

Comment:
btw, did you have to generate this yourself or do we have a less manual way of doing these?

Reply:
It's all manual right now. I have some code that generates the list of patches automatically, but it's only used for testing purposes. It should be easy to make it into a separate tool to generate these.